testing/sdk_install.py

```diff
@@ -102,6 +115,9 @@ def uninstall(
         except (dcos.errors.DCOSException, ValueError) as e:
             log.info('Got exception when uninstalling package, ' +
                           'continuing with janitor anyway: {}'.format(e))
+            if 'marathon' in str(e):
```

Comment:
Yep, it's actually a package describe call that is made before the package uninstall call.